src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,18 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	
+	private final String block;
+	
+	public Block(String str) {
+		this.block = str;
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.